utility/stream_limiter_ban_email.py

```diff
@@ -295,9 +296,9 @@ def send_notification(to=None, friendly=None, val_cnt=None, val_tot=None, mess=N
         try:
             if act_lst.count(user) >= LIMIT:
                 # Trigger for first and next violation
-                unshare(user) # Remove libraries
-                share(user, BAN) # Share banned library
-                sys.stdout.write("Shared BAN_LIBRARY with user {0}".format(i))
+                unshare(user)  # Remove libraries
+                share(user, BAN)  # Share banned library
+                sys.stdout.write("Shared BAN_LIBRARY with user {0}".format(user))
```

Comment:
`i` wasn't defined, from the context it appears that `user` was the intended variable.